src/com/github/TylerWhitmore/cs3230/StringOperations.java

```diff
@@ -0,0 +1,29 @@
+package src.com.github.TylerWhitmore.cs3230;
+import java.util.List;
+import java.util.ArrayList;
+
+public class StringOperations {
+    public static List<Boolean> palindrome(List<String> str){
+        List<Boolean> tf = new ArrayList<Boolean>();
+        for(int i = 0; i < str.size(); i++){
+            String curr = str.get(i); //holding current string
+            if(curr.length() > 1){ //more than one character
+                StringBuilder manipulating = new StringBuilder();
+                manipulating.append(str); //add str to the manipulated variable
```

Comment:
You have a bug here. This line has a simple error. Try running your code with the debugger and see what you `str` is.